### ISSUE TITLE
[VOTE] Avoid Perl-style comments

### ISSUE THREAD

**Issue by budrea** [shoppingflux/coding-style-php#23]
### Summary

Should we avoid Perl-style comments ? We use them throughout the code, but rarely. I would suggest to stick to PHP-style comments. 

### Valid Code

```php
<?php

// good single line comment

/*
good multi-line comment
*/

/**
* php-doc style comment is ok
*/

```

### Invalid Code

```php
<?php

# Perl-style comment (should be avoided)
```

**Comment by budrea** [comment]
Since the issue I tried to solve in [this PR](https://github.com/shoppingflux/coding-style-php/pull/16)  does not seem to be unanimously accepted, I've opened this PR so we can all vote on this issue.

I'll take into account the results and either move forward or close the previously cited PR.

Please vote : @ronan-gloo , @mdumoulin , @ddattee , @olivierh94 , @yvanngn

**Comment by ronan-gloo** [comment]
closed by Perl-style comments are not allowed #16

**Comment by ronan-gloo** [comment]
Pour courage pour les prochains csfixes ^^
